docs/specifications/tests/DNSSEC-TP/dnssec01.md

```diff
@@ -5,19 +5,23 @@
 
 ## Objective
 
-The list of allowed Digest Algorithms in a DS record published by 
-the parent is specified by [RFC 8624], section 3.3, and is published 
-in the [IANA registry] of *DS RR Type Digest Algoritms*. No DS 
+The list of allowed Digest Algorithms in a DS record published by
+the parent is specified by [RFC 8624], section 3.3, and is published
+in the [IANA registry] of *DS RR Type Digest Algoritms*. No DS
 Digest Algorithm values, other than those specified in the RFC and
 allocated by IANA, should be used in public DNS.
 
-If [RFC 8624] and the [IANA registry] disagree on the same DS digest 
-algorithm, the RFC takes precedence until the registry has a been 
+If [RFC 8624] and the [IANA registry] disagree on the same DS digest
+algorithm, the RFC takes precedence until the registry has a been
 updated with a reference to the RFC.
 
-The table of algorithms below is for reference only and is copied from [IANA 
+At the time of writing (2020-04-08), algorithm 1 (SHA-1) is still
+in wide use even though it is no longer considered to be secure
+([Wikipedia]).
```

Comment:
Is Wikipedia a reliable reference?

Reply:
Yes, Wikipedia has good checks of the content.